enhancements/multi-arch/heterogeneous-architecture.md

```diff
@@ -0,0 +1,328 @@
+---
+title: heterogeneous-architecture-clusters
+authors:
+  - "@Prashanth684"
+reviewers:
```

Comment:
we will want the reviewer section to include every team lead of a team that is impacted by this (e.g. console, image registry, ART, etc) with a quick sentence explaining which piece of this impacts them (so they can focus their review on that section and not read the whole thing)